src/components/PatientRecord/PatientRecord.tsx

```diff
@@ -155,6 +157,7 @@ const Visualizer: FC<VisualizerProps> = ({ resourceType, resourcesByType }) => {
   const patient = usePatient().patient as fhir.Patient;
 
   if (resourceType === 'Pathway') return <PathwayVisualizer />;
+  else if (resourceType === 'Mcode') return <McodeVisualizer />;
```

Comment:
The way `Mcode` and `Pathway` are handled in this file is confusing and I had to look at it for a bit to realize what was going on.  It suggests that they are `resourceTypes` just like FHIR resources.  I know previous PRs have discussed ways to reduce the number of `if`s in this file, but even just the naming is confusing to understanding what's going on.

I think we should at least rename `resourceTypes` on line 31 to `recordSections`.